### PR TITLE
fix(infra): getting rid of sentry interceptor

### PR DESCRIPTION
**Pull request by pshomov**
# ...

Seems it is using a lot of resources when reporting issues to its central service as found out by @eirikurn .

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/user-profile/src/main.ts [hunk 1]
```diff
@@ -1,19 +1,11 @@
 import { bootstrap } from '@island.is/infra-nest-server'
 import { AppModule } from './app/app.module'
 import { openApi } from './openApi'
-import * as Sentry from '@sentry/node'
 import { environment } from './environments'
-import { SentryInterceptor } from '@island.is/infra-monitoring'
-
-Sentry.init({
-  dsn: environment.sentry.dsn,
```
Comment (eirikurn) [inline apps/services/user-profile/src/main.ts:9]: Get rid of this environment field and infra secret as well?

File: apps/air-discount-scheme/backend/src/main.ts [hunk 2]
```diff
@@ -1,21 +1,12 @@
 import { bootstrap } from '@island.is/infra-nest-server'
-import * as Sentry from '@sentry/node'
-import { SentryInterceptor } from '@island.is/infra-monitoring'
 
 import { AppModule } from './app/app.module'
-import { environment } from './environments'
 import { openApi } from './openApi'
 
-Sentry.init({
-  dsn: environment.sentry.dsn,
```
Comment (eirikurn) [inline apps/air-discount-scheme/backend/src/main.ts:10]: Same here

**Review by eirikurn — CHANGES_REQUESTED** [review]

File: apps/services/user-profile/src/main.ts (hunk 1 above)
Comment (pshomov) [reply apps/services/user-profile/src/main.ts:9]: fixed

File: apps/air-discount-scheme/api/src/main.ts [hunk 3]
```diff
@@ -1,18 +1,9 @@
 import { bootstrap } from '@island.is/infra-nest-server'
-import * as Sentry from '@sentry/node'
 
-import { SentryInterceptor } from '@island.is/infra-monitoring'
 import { AppModule } from './app/app.module'
-import { environment } from './environments'
-
-Sentry.init({
-  dsn: environment.sentry.dsn,
```
[the remark this answers is not in the document]
Comment (pshomov) [reply apps/air-discount-scheme/api/src/main.ts:9]: fixed

**Review by Snojo — APPROVED** [review]
Looks good after fixes

**Review by rafnarnason — APPROVED** [review]
LGTM 🍕

**Review by Valur — APPROVED** [review]

**Pull request merged**